src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -590,6 +590,16 @@
 1 Format Parameter:
 1. failure: The failure message from validation</comment>
   </data>
+  <data name="LogFormatString_QueueDoesNotExist" xml:space="preserve">
+    <value>The queue {0} does not exist; ensure that the queueName and queueAccountUri fields are set correctly.</value>
```

Comment:
nit: We might want to put in holes for `queueName` and `queueAccoutUri` and then fill them in programmatically with constants (e.g. nameof(...)) so that localization doesn't accidentally localize these.